styles/main.css

```diff
@@ -1,31 +1,32 @@
 @charset "utf-8";

```

Comment:
I've made a number of formatting changes to your CSS code as things were starting to look a bit messy here.